### PR TITLE
MEN-4030: Add test for DBus busconfig file.

### PR DESCRIPTION
**Pull request by kacf**

### CODE REVIEW

**Comment by kacf** [comment]
The revert was discussed with @lluiscampos and agreed upon.

File: tests/test_rootfs.py [hunk 1]
```diff
@@ -116,14 +116,32 @@ def test_expected_files_ext234(
...
+                        "debugfs",
+                        "-R",
+                        "ls -l -p /usr/share/dbus-1/system.d",
+                        latest_rootfs,
+                    ],
+                    cwd=tmpdir,
+                ).decode()
+                assert any(
+                    [
+                        line.split("/")[5] == "io.mender.conf"
```
Comment (tranchitella) [inline tests/test_rootfs.py:131]: This needs to be updated because of https://github.com/mendersoftware/mender/pull/658

Comment (kacf) [reply tests/test_rootfs.py]: Ah, good catch, fixed!

File: tests/test_rootfs.py [hunk 2]
```diff
@@ -54,7 +54,7 @@ def verify_fstab(data):
             occurred[cols[1]] = True
 
     @pytest.mark.only_with_image("ext4", "ext3", "ext2")
-    @pytest.mark.min_mender_client_version("1.0.0")
+    @pytest.mark.min_mender_version("1.0.0")
     def test_expected_files_ext234(
```
Comment (lluiscampos) [inline tests/test_rootfs.py:58]: As we are reusing an existing test, we need to bump `min_mender_version` to 2.5.0

Comment (lluiscampos) [reply tests/test_rootfs.py:58]: Alternatively we could have a separate test, but I have no inconvenient in reusing and skipping this test for upcoming bugfix releases, as this is very low risk.

**Review by lluiscampos — CHANGES_REQUESTED** [review]

File: tests/test_rootfs.py (hunk 2 above)
Comment (kacf) [reply tests/test_rootfs.py:58]: Good catch, will fix!

**Review by lluiscampos — APPROVED** [review]

**Pull request merged**